examples/design-system/package.json

```diff
@@ -1,21 +1,19 @@
 {
   "private": true,
   "scripts": {
-    "build": "turbo run build",
```

Comment:
We should keep `run` in these. The general advice from @nathanhammond has been to not use shorthand syntax when putting turbo commands in scripts. I generally agree.